### PR TITLE
Feature/id type not attributes

### PR DESCRIPTION
**Pull request by crbrox**
Reject VRs and notices with an attribute named 'id' or 'type'

@fgalan 
@dmoranj 

FYI: @iariasleon

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -4,4 +4,4 @@ Change HA model for execution of actions
 Fix error if parameter "type" exist in Update entity action (#30)
 Added 'twitter' action
 Changed 'update' action, any entity
-
+Do not allow 'id' and 'type' as attribute names
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: The corresponding .md file should explain that id and type are forbidden as attribute names.

File: test/data/bad_notices/notice_type_as_attr.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (fgalan) [inline test/data/bad_notices/notice_type_as_attr.json]: I see two .json for test the "bad condition" in VR API. What abotu a json to test it in the plain rules API?

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Fixed ba2e3dcf494252

File: test/data/bad_notices/notice_type_as_attr.json (hunk 2 above)
Comment (fgalan) [inline test/data/bad_notices/notice_type_as_attr.json]: Fixed in f207c48

**Comment by fgalan** [comment]
LGTM ( passing the ball to @dmoranj for the final LGTM and merge)

File: lib/models/actions.js [hunk 3]
```diff
@@ -37,6 +37,44 @@ var util = require('util'),
...
+
+function validateAction(axn) {
+    //There is an action
+    if (typeof axn.type !== 'string') {
+        return new errors.MissingActionType(axn);
+    }
+
+    //Action type
+    switch (axn.type) {
+    case 'email':
```
Comment (dmoranj) [inline lib/models/actions.js]: Does this indentation work with the linter?

Comment (crbrox) [inline lib/models/actions.js]: Apparently, in my local environment, it does pass

Comment (dmoranj) [inline lib/models/actions.js]: OK. NTC.

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**